config/services.example.json

```diff
@@ -34,9 +34,49 @@
         "error_params": {
           "scaling": 2,
           "min_th_up": 0.1,
-          "min_th_low": -0.1,
+          "min_th_low": -0.1
+        }
+      }
+    }
+  },
+  "SeasonalDecompositionEnsemble": {
+    "scheduler_options": {
+      "interval_secs": 30,
+      "plugin": "PollSeasonalDecompositionEnsemble",
```

Comment:
It isn't. The plugin is a scheduler option. You could use a PollTask or the plugin itself.